Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -28,6 +28,40 @@ def edit_args
     end
   end
 
+  sig { void }
+  def fail(path, cask)
+    name = path.basename(".rb").to_s
+
+    if (tap_match = Regexp.new(HOMEBREW_TAP_DIR_REGEX.source + /$/.source).match(path.to_s))
```

Comment:
I'd use interpolation on the Regex instead of `+`

Reply:
Done.